api/egl.xml

```diff
@@ -791,7 +791,9 @@
         <enum value="0x3350" name="EGL_GL_COLORSPACE_SCRGB_LINEAR_EXT"/>
         <enum value="0x3351" name="EGL_GL_COLORSPACE_SCRGB_EXT"/>
         <enum value="0x3352" name="EGL_TRACK_REFERENCES_KHR"/>
-            <unused start="0x3353" end="0x335F"/>
+            <unused start="0x3353" end="0x3356"/>
+        <enum value="0x3357" name="EGL_CONTEXT_PRIORITY_REALTIME_NV"/>
```

Comment:
Any reason to add it in the middle of this free range?
No objection, it's your block, I'm just curious :)

Reply:
Enum Values from 0x3353-0x3356 are being used for one of our internal extensions, which we've not made public yet, so just using next free available from our internal range.